web_src/less/themes/theme-arc-green.less

```
@@ -1593,3 +1593,7 @@ footer .container .links > * {
 .tribute-container li:hover {
     background: #728e5e !important;
 }
+
+.repository .repo-header .ui.huge.breadcrumb.repo-title .repo-header-icon .avatar {
```

```suggestion
.repo-header-icon .avatar {
```

Sometimes these selector chains in Less get quite crazy. This should be enough. I suggest you extract the rule in the `_repository.less` and put it at the end of the file without any crazy nesting.

---

Might want to override the behavior or size of avatar if used in different place however?

---

On second thought, can't do it actually because more specific `.ui.avatar.image` from Fomantic will override size and margin

---

Ah, `.avatar` is probably used elsewhere, right? In that case we may need the nesting for the added specificity without having to resort to `!important`. Generally, I do not recommend nesting less more than around 3 level, thought.